_data/communication.yml

```
@@ -129,6 +129,12 @@ websites:
       - sms
     doc: https://help.groupme.com/hc/en-us/articles/360000579167
 
+  - name: Groups.io
+    url: https://groups.io
+    img: groups_io.png
+    tfa:
+      - totp
+
```

```suggestion
doc: "/notes/groups.io/"

```
As there is no documentation by groups.io for setting up 2FA, I think that we should include steps to set it up. See [twofactorauth/notes/](https://github.com/2factorauth/twofactorauth/tree/master/notes)

The steps could be: 
1. [Login](https://groups.io/login) to your account. 
2. Click your email address in the top right-hand corner and click _account_. 
3. Click _security_. 
4. Click _set up two-factor authentication_. 
5. Enter your password and click _enable_. 
6. Follow the steps to enable 2FA. 

Or at least link to [this comment](https://github.com/2factorauth/twofactorauth/pull/4320#issuecomment-560177656).

---

According to https://github.com/2factorauth/twofactorauth/pull/4199#issuecomment-546748150 the notes should not be necessary.

---

Yeah no need for documenting that.